### DISCUSSION TITLE
Color based on one field, display another field

### DISCUSSION THREAD

**Discussion by courtalj** [grafana/grafana#53348] [Dashboards, Panels & Visualizations]
In our use case, the "alert" status for a particular field is included in the database, and we want to display the value while coloring it based on the separate alert field. Our alert status is literally a string "GREEN", "YELLOW_LOW", "RED_LOW", "YELLOW_HIGH", "RED_HIGH". Both the value and alert fields can be retrieved in a single query, but currently it seems that colors can only be applied based on the field that is displayed.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
